packages/block-editor/src/components/global-styles/filters-panel.js

```diff
@@ -75,6 +90,36 @@ const DEFAULT_CONTROLS = {
 	duotone: true,
 };
 
+const popoverProps = {
+	placement: 'left-start',
+	offset: 36,
+	shift: true,
+	className: 'block-editor-duotone-control__popover',
+	headerTitle: __( 'Duotone' ),
+};
+
+const LabeledColorIndicators = ( { indicators, label } ) => (
+	<HStack justify="flex-start">
+		<ZStack isLayered={ false } offset={ -8 }>
+			{ indicators.map( ( indicator, index ) => (
+				<Flex key={ index } expanded={ false }>
+					{ indicator === 'unset' || ! indicator ? (
+						<ColorIndicator className="block-editor-duotone-control__unset-indicator" />
+					) : (
+						<DuotoneSwatch values={ indicator } />
+					) }
+				</Flex>
+			) ) }
+		</ZStack>
+		<FlexItem
+			className="block-editor-panel-color-gradient-settings__color-name"
```

Comment:
Another case where it seems we're reusing a classname? IT might indicate that we're missing a UI component to reuse between the different places.